extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -271,17 +287,29 @@ private function appendRecordsFromResponse( array &$records, $response, array &$
 				continue;
 			}
 
-			$wikiInfo = WikiFactory::getWikiByID($site);
+			if ( !isset($domainCache[$site]) ) {
```

Comment:
Minor: There's a number of white space violations in this block. Function calls should have padding in them. Eg
```php
// bad 
if ( !isset($domainCache[$site]) )

// good
if ( !isset( $domainCache[$site] ) )
```